spec/support/mock_model.rb

```diff
@@ -1,2 +1,5 @@
 class MockModel
```

Comment:
Seemed to me that both were used for the tests, I chose randomly. But it seems that MockModel it is used in [only one test](https://github.com/thoughtbot/paperclip/blob/master/spec/paperclip/url_generator_spec.rb#L46). Don't merge, I prepare two new pull request. One with the fixed code for issue #1453, another to remove completely `MockModel`, is useless(right?).